src/net/toxme.cpp

```diff
@@ -78,14 +134,20 @@ QByteArray Toxme::prepareEncryptedJson(int action, QString payload)
 ToxId Toxme::lookup(QString address)
 {
     // JSON injection ?
+    address = address.trimmed();
     address.replace('\\',"\\\\");
     address.replace('"',"\"");
 
     ToxId id;
     const QString json{"{\"action\":3,\"name\":\""+address+"\"}"};
     static const QByteArray pattern{"public_key\""};
```

Comment:
That's interesting, I tested it today and it returned "tox_id". That's also what the API says: https://github.com/subliun/toxme/blob/master/doc/api.md